Tools/autotest/sim_vehicle.py

```diff
@@ -1250,7 +1242,7 @@ def generate_frame_help():
     if cmd_opts.no_mavproxy:
         time.sleep(3)  # output our message after run_in_terminal_window.sh's
         progress("Waiting for SITL to exit")
-        wait_unlimited()
+        signal.pause()
```

Comment:
we cannot use that as windows don't support it

Reply:
Okay, I'll revert that change.